doc/api/esm.md

```diff
@@ -577,8 +577,68 @@ console.log(five); // Logs `5`
 node b.mjs # works
 ```
 
-<i id="esm_experimental_loaders"></i>
+## HTTPS and HTTP imports
+
+> Stability: 1 - Experimental
+
+Importing network based modules using `https:` and `http:` is supported under
+the `--experimental-https-modules` flag. This allows web browser-like imports to
+work in Node.js with a few differences due to application stability and security
+concerns that are different when running in a privileged environment instead of
+a browser sandbox.
+
+### Imports are limited to HTTP/1
+
+Automatic protocol negotiation for HTTP/2 and HTTP/3 is not yet supported.
+
+### HTTP is limited to loopback addresses
+
+`http:` is vulnerable to man in the middle attacks and is not allowed to be used
+for addresses outside of the IPv4 address 127.0.0.1 and the IPv6 address ::1 .
+Support for `http:` is intended to be used for local development.
+
+### Authentication is never sent to the destination server.
+
+`Authorization`, `Cookie`, and `Proxy-Authorization` headers are not sent to the
+server. Avoid including user info in parts of imported URLs. A security model
+for safely using these on the server is being worked on.
+
+### CORS is never checked on the destination server
+
+CORS is designed to allow a server to limit the consumers of an API to a
+specific set of hosts. This is not supported as it does not make sense for a
+server-based implementation.
 
+### Cannot load non-network dependencies
+
+These modules cannot access other modules that are not over `http:` or `https:`.
+Usage of parameters and initializers can get around this:
```

Comment:
This feels a little overly restrictive to me. Basically it rules out a lot of use cases. Ideally if we have a more fine-grained permissions model, it would be possible to scope what "scopes" can import what protocols / hosts / paths.

Reply:
Policies can be used for exceptional cases and are the way to give permissions to resolution generally.

Reply:
I wouldn't call these cases exceptional I would call them the norm. JSPM Node.js import maps rely on this significantly.

It seems arbitrary to make this policy decision over any other.... ideally policies can tackle these restrictions generally to begin with making this an easy "one line rule" to add.

Reply:
The concern is how radical transitive dependencies can be intercepted or change ownership. I don't think it is arbitrary. With opt-in you avoid having this problem be introduced accidentally. Even with import maps they are similar to policies in that they override normal resolution behavior. The goal is to allow such imports across protocols, not to do so without any differences from the web which has a different set of security concerns. Adding a single line could in theory be possible with patterns if we add them, but still likely shouldn't occur since arbitrary access to files and core are exactly what is mitigated.

Reply:
I would argue this is in fact the primary feature of policy overall - in being able to delegate which URL "scopes" have which core package permissions. As such I hope we can treat it as a very natural thing for policy to do, and that the default scenario of "allow all" shouldn't require explicitly and manually adding 100s of items, while still putting security first.

Reply:
The default when using a policy must not be to allow all.

Reply:
Would policy scopes for external URLs permit allowing core modules explicitly? Eg like:

```
"scopes": {
    "https://cdn.com/": {
      "dependencies": {
        "fs": true
      }
    }
  }
```